arch/arm/arm64/Config.uk

```diff
@@ -54,3 +54,19 @@ config ARM64_ERRATUM_858921
 	  This option enables a workaround for Cortex-A73 (r0p0 - r0p2),
 	  whose counter may return a wrong value when the counter crosses
 	  a 32-bit boundary. The newer Cortex-A73 are not affected.
+
+config ARM64_ERRATUM_835769
```

Comment:
Just a nitpick: The descriptions are a bit inconsistent. One describes how the erratum is fixed, while the other one just states that it is fixed by passing a flag to GCC (which is also true for the first one). Moreover, as both errata are handled by GCC, in theory their implementation can change, so it would be better to avoid elaborating on their implementation and instead use something along the lines of "This flag enables GCC's workaround for the ARM Cortex-A53 erratum number xyz".